grow/preprocessors/google_drive.py

```diff
@@ -257,6 +257,8 @@ def download(cls, spreadsheet_id, gids=None, format_as='list', logger=None,
                             row = row[1:]
                             row_len = len(row)
                             for col, grid_key in enumerate(headers):
+                                if not grid_key:
```

Comment:
Wonder if we should also exclude keys that start with `_`? This is opinionated but it might be helpful? Not sure.

Reply:
Good idea. Updated.